content/blog/2019/02/2019-02-08-windows-installers.adoc

```diff
@@ -0,0 +1,109 @@
+
+ 1. The installer will only support 64-bit systems going forward. This is the vast majority of Windows systems these days, 
+    so this will help more users install Jenkins using the installer package.
+ 2. The user is now able to enter user information for the service and select the port that Jenkins will use and verify that the port is available.
+ 3. The installer no longer bundles a JRE, but will search for a compatible JRE on the system. If the user wants to use a different JRE, they can specify during install.
+ 4. The installer has support for running with a Java 11 JRE, including the components listed on the link:https://jenkins.io/blog/2018/12/14/java11-preview-availability/[Java 11 Preview Page].
+ 5. the JENKINS_HOME directory is placed in the LocalAppData directory for the user that the service will run as, this aligns with modern Windows file system layouts.
+ 6. The installer has been updated with branding to make it look nicer and provide a better user experience. 
+ 
+## Screenshots
+
+Below are screenshots of the new installer sequence:
+
+### Step 1
+
+image:/images/post-images/2019-02-08-windows-installer/new_installer_1.png[Installer Startup, role=center]
+
+The Jenkins logo is now a prominent part of the UI for the installer.
+
+### Step 2
+
+image:/images/post-images/2019-02-08-windows-installer/new_installer_2.png[Installation Directory, role=center]
+
+The Jenkins logo and name are now in the header during all phases of the installer.
+
+### Step 3
+
+image:/images/post-images/2019-02-08-windows-installer/new_installer_3.png[Account Selection, role=center]
+
+The installer now allows you to specify the username/password for the account to run as and checks that the account has LogonAsService rights.
+
+### Step 4
+
+image:/images/post-images/2019-02-08-windows-installer/new_installer_4.png[Port Selection, role=center]
+
+The installer also allows you to specify the port that Jenkins should run on and will not continue until a valid port is entered and tested.
+
+### Step 5
+
+image:/images/post-images/2019-02-08-windows-installer/new_installer_5.png[JRE Selection, role=center]
+
+Instead of bundling a JRE, the installer now searches for a compatible JRE on the system (JRE 8 is the current search). If you want to use a different
+JRE on the system than the one found by the installer, you can browse and specify it. Only JRE 8 and JRE 11 runtimes are supported. The installer will
+automatically add the necessary arguments and additional jar files for running under Java 11 if the selected JRE is found to be version 11.
+
+### Step 6
+
+image:/images/post-images/2019-02-08-windows-installer/new_installer_6.png[Install It, role=center]
+
+All of the items that users can enter in the installer should be overridable on the command line for automated deployment as well. The full list of properties that
+can be overridden will be available soon.
+
+## Next Steps

```

Comment:
Add some contributing guidelines?

Reply:
I can do that.

Reply:
Anyway, there is a Gitter link. People will know how to reach out